homework8_subsequence.py

```diff
@@ -0,0 +1,36 @@
+"""Module providing a function printing python version."""
+
+
+def test_for_increasing_function(arr):
+    """Testing for increasing function."""
+    for i in range(len(arr) - 1):
+        if arr[i] >= arr[i + 1]:
+            return False
+    return True
+
+
+def will_it_be_strictly_increasing(arr):
+    """The function slices through and checks on increasing."""
+    if test_for_increasing_function(arr):
+        return True
+
+    for i in range(len(arr)):
+        new_arr = arr[:i] + arr[i + 1:]
+        if test_for_increasing_function(new_arr):
+            return True
+    return False
+
+
+def answer(arr):
+    """Response output."""
+    if will_it_be_strictly_increasing(arr):
+        print("Answer : True")
+    else:
+        print("Answer : False")
+
+
+answer([1, 2, 3])
```

Comment:
and here if you return bool type, so you can use assert here to check that your code works as expected

Reply:
Done